app/helpers/physical_storage_helper/textual_summary.rb

```diff
@@ -97,4 +97,16 @@ def textual_rack_name
   def textual_lowest_rack_unit
     {:label => _("Lowest Rack Unit"), :value => @record.asset_detail&.lowest_rack_unit}
   end
+
+  def textual_capabilities
+    {:label => _("Capabilities"), :value => parse_capabilities(@record.physical_storage_family&.id)}
+  end
+
+  def parse_capabilities(storage_family_id)
+    capabilities = []
+    if storage_family_id
+      capabilities = PhysicalStorageFamily.find(storage_family_id)&.capabilities
+    end
```

Comment:
Two things:
1. Prefer setting the physical_storage capabilities to be equal to the physical_storage_family.capabilities in your refresh parser so we only have to hit one location here to show them.  Consider someone using the API where this logic won't exist (and shouldn't have to exist)
2. If you were going to reference the storage_family for some reason you should use the assoication rather than doing a manual query (e.g. `@record.physical_storage_family&.capabilities` don't actually do this here, but if you had to in the future use the assoc not a manual query)